doc/API.md

```diff
@@ -29,6 +30,25 @@ Note that all addresses are bech32-encoded *version 0* native Segwit `scriptPubK
 
 ## General
 
+### `help`
+
+Display all available commands.
+
+#### Response
+
+| Field      | Type  | Description                                 |
+| ---------- | ----- | ------------------------------------------- |
+| "commands" | array | Array of [help resource](#Help_resource)    |
```

Comment:
You should use backquotes to have `commands` be rendered as pre-formatted. Also the link is wrong (it was copy pasted from my example which was just an instance. I tested and it should be `(#help-resource)`.

Reply:
Yeah, I missed that!
Made the required changes